eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java

```diff
@@ -42,11 +42,11 @@
  */
 public class EidpLoginModule implements LoginModule {
 
-  private Subject subject;
+//  private Subject subject;
```

Comment:
\[Checkstyle\] WARNING: Comment has incorrect indentation level 0, expected is 2, indentation should be the same level as line 46\.